CITATION.cff

```diff
@@ -0,0 +1,9 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: Prochaska, Hennawi, Westfall, Cooke, Wang
+    given-names: J. Xavier, Joseph, Kyle, Ryan
```

Comment:
It seems we either need an extra `given-name`, or we have an extra `family-name`

Reply:
edited